libfaketime/__init__.py

```diff
@@ -103,17 +104,22 @@ def __init__(self, datetime_spec, only_main_thread=True):
             _datetime = dateutil.parser.parse(datetime_spec)
 
         self.time_to_freeze = _datetime  # freezegun compatibility
-        self.libfaketime_spec = _datetime.strftime('%Y-%m-%d %T %f')
 
     def _should_fake(self):
         return self.only_main_thread and threading.current_thread().name == 'MainThread'
 
+    def _format_datetime(self, _datetime):
+        return _datetime.strftime('%Y-%m-%d %T %f')
+
+    def tick(self, delta=datetime.timedelta(seconds=1)):
+        self.time_to_freeze += delta
+        os.environ['FAKETIME'] = self._format_datetime(self.time_to_freeze)
```

Comment:
Hm, I think this may have the same behavior I'm confused about:
```
fake_time(...).tick()
# time is now faked, but start/enter was never called
```

Is this the same way freezegun works, or is tick() a noop for it until start/enter are called? I know it's a weird edge case; I just want to try and match behavior.